src/fetchData/util.ts

```diff
@@ -0,0 +1,43 @@
+import { resourceNameArray, Resources } from '../utils/response/apiResponse';
+import { RawApiResponse } from '../utils/response/rawApiResponse';
+import { ResourceResponse } from '../utils/response/resourceResponse';
+import { FetchDataResponse } from './type';
+
+export function createFetchDataResponse(rawApiResponse: RawApiResponse): FetchDataResponse {
```

Comment:
put the file please to the same folder where `createDeltaSyncResponse` is. it is `js-client-library/src/utils/response`